scripts/test-projects-workspace/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "test-projects-workspace"
+description = "Build project workspace with `forc-pkg`"
+version = "0.1.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+publish = false
+
+[dependencies]
+forc-pkg = "0.32.2"
+forc-util = "0.32.2"
+#forc-fmt = "0.32.2" todo do we need this
+walkdir = "2.3.2"
+glob = "0.3.1"
+
+termcolor = "1.1"
+tokio = { version = "1.21.2", features = ["process", "macros", "rt", "rt-multi-thread"] }
+futures-util ="0.3.24"
+clap = { version ="4.0.15", features =["cargo", "color", "derive"] }
```

Comment:
I would keep the CLI interface we had `test_projects {build, format etc.}` If we agrree that we do not need that anymore than you can remove `clap` as a dependency.